README.md

```diff
@@ -11,6 +11,24 @@ It depends on the [LS Server Core](https://github.com/NYPL-Simplified/server_cor
 * [How to set up a development environment](https://github.com/NYPL-Simplified/Simplified/wiki/Deployment-Instructions)
 * Two sets of Ansible playbooks are in development: [One developed by Minitex](https://github.com/Minitex/ansible-playbook-libsimple) and [a derivative developed by Amigos Library Services](https://github.com/alsrlw/ansible-playbook-libsimple)
 
+## Generating Documenation
+
+Code documentation using Sphinx can be found on [readthedocs.org](). It currently documents this repo's `api` directory, `scripts` file, and the `core` directory submodule. The configuration for the documentation can be found in `/docs`.
+
+There are two steps to building the documentation: generate the `.rst` files from the codebase and generate the html static site (not to be committed). Inside the `/docs` directory, run:
+
+```
+   $ sphinx-apidoc -f -o source/ .. ../tests/* ../core/tests/* ../integration_tests/*
+```
+
+```
+   $ make html
+```
+
+The `sphinx-apidoc` command will generate a new set of `.rst` files. Currently, there is no automatic way to generate them using continuous integration so they will need to be generated manually.
```

Comment:
Is it possible to have something that runs during CI to ensure that the docstrings are formatted properly, so we don't inadvertently break the docs?

Reply:
pylint is the obvious candidate, but I don't see anything that does exactly this. By default pylint will complain about _missing_ docstrings, and you can [get it to complain](https://docs.pylint.org/en/1.6.0/extensions.html) about missing _parameters_ in docstrings, but we're in the middle and I don't see a way to do that.

We could also run sphinx during CI. We'd need to have a way of running it just on core. Otherwise we wouldn't find out about a core problem until we brought that revision into circulation.

Reply:
I have also not found a way to deploy to readthedocs from Travis CI and there are no webhooks for it. So when a push is made, both readthedocs and Travis build and deploy separately.

Reply:
That's fine; I'm only suggesting we run sphinx because sphinx is the piece of software that can catch these particular errors.